tests/test_validators.py

```diff
@@ -72,6 +74,23 @@ def test_public_email_domain(self):
             self.assertIn('webmail_domain', self.webmail_form.errors)
             self.assertNotIn('not_webmail_domain', self.webmail_form.errors)
 
+    def test_public_email_domain_helper(self):
+        with self.app.test_request_context('/'):
+            # As no default is provided and the domain lookup will trigger MXLookupExceptiom
+            # the helper function will raise as expection
```

Comment:
Comment could be better:

`Intentionally trigger a DNS lookup failure using an invalid domain name. Since no default is provided, we will receive an exception`